bundle/metadata/annotations.yaml

```diff
@@ -4,12 +4,13 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: nvidia-network-operator
-  operators.operatorframework.io.bundle.channels.v1: 23.10.0-1
-  operators.operatorframework.io.bundle.channel.default.v1: 23.10.0-1
+  operators.operatorframework.io.bundle.channels.v1: stable,v24.4.1
```

Comment:
after discussing this with freddy, we would like to have the following channels:

1. stable as we have today
2. dedicated channel for a given network operator major release: e.g v24.7 where all minor updates will refer to in addition to the stable channel. this will align channel management with what gpu operator are doing.

this scheme will be applied from v24.7 onwards. so when building v24.7.0 bundle (manually for now until automation is in place) we will have a stable channel and a v24.7 channel.